app/models/user.rb

```diff
@@ -24,8 +24,8 @@ class User < ActiveRecord::Base
   scope :donors,      -> { where( permission_id: nil ) }
   scope :reviewers,   -> { where( permissions: { name: 'Reviewer'   } ).joins(:permission) }
   scope :supervisors, -> { where( permissions: { name: 'Supervisor' } ).joins(:permission) }
+  scope :system,      -> { where( permissions: { name: 'System' } ).joins(:permission) }
```

Comment:
Line is too long. [88/80]<br>Unnecessary spacing detected.<br>Space inside parentheses detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.